src/add-hooks.coffee

```diff
@@ -95,23 +94,43 @@ addHooks = (runner, transactions, callback) ->
         callback(new Error(msg))
     else
 
-      # no data found, doing nothing
+      # No data found, doing nothing
       return callback()
 
   # Loading hookfiles from fs
   else
+    # Expand hookfiles - sort files alphabetically and resolve their paths
+    hookfiles = [].concat runner.configuration?.options?.hookfiles
+    files = hookfiles.reduce((result, item) ->
```

Comment:
I'd prefer better variable name then `item` - e.g. `path` or `file` or something specific like that.